### DISCUSSION TITLE
META AND DEFİ

### DISCUSSION THREAD

**Discussion by alparslanmeva** [IBAX-io/go-ibax#472] [Ideas] [answered]
### Clear and concise description of the problem

günümüz meta evren dünyasında söz sahibi olabilecek projeler ile ibax ın kendi özel agından kaliteli projeler üretilip bunu defi ve playto earn oyunlarıyla birleştirilip mükemmel ötesi bir dünya yaratılabilir diye düşünüyorum.ayrıca gercek ve gercek olmayan kişilerinde kimlik onayları alınaraktan airdroplardan adaletli bir şekilde faydalandırılmasını istiorum.yaptıklarınızla ve yapmak istediklerinizle güzel yolda ilerliyorsunuz başarılarınızın devamını diliyorum.

### Suggested solution

META + DEFİ +PLAY TO EARN = İBAX CHAİN :)

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ibaxio** [comment, marked as answer]
We already have such a plan, thank you for your attention and support, you can also invite friends together to IBAX Network to create your own ecosystem.
